main_test.go

```diff
@@ -57,6 +57,7 @@ func TestBuild(t *testing.T) {
 		"json.go",
 		"map.go",
 		"math.go",
+		"mapgrowth.go",
```

Comment:
Please keep this list sorted.
Also, why not merge mapgrowth.go with map.go? Unfortunately, compilations are still relatively slow and adding new tests slows down everything a lot.

Reply:
It's a different file because I just added the test program I was using for development and dropped it into the testdata directory.  I'll add it to the `map.go` file properly.